### PR TITLE
Defined REST API for configuration reload

### PR DESCRIPTION
**Pull request by maheshkelkar**
References #120

### CODE REVIEW

**Comment by maheshkelkar** [comment]
@rtyler @GiriDandu Could you please review this?

File: features/api/updating.feature [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Feature: DeployDB UPDATE APIs
+
+  As a RESTful client or user
+  I should be able to update DeployDB configuration
+
+  @wip
+  Scenario: Reloading the config while system is idle will succeed
+
+    Given configuration directory is "config"
```
Comment (GiriDandu) [inline features/api/updating.feature]: Can we have this /config/reload that way in the future we can have /config/model/reload or /config/model/list. May be its too far fetched idea

Comment (GiriDandu) [inline features/api/updating.feature]: Also you may want to move this under /config/updating.feature from /api/updating.feature

**Comment by rtyler** [comment]
I think we need to find a way to get this onto the admin servlet, but the API is fine as far as I'm concerned right now

**Pull request merged**